document/FileFormat.mdk

```diff
@@ -913,16 +941,28 @@ An `invocation` object may contain a property named `commandLine` whose value is
 containing the runtime parameters with which the tool was invoked.
```

Comment:
>was invoked [](start = 54, length = 11)

need to clearly specify whether the .exe or script name is provided as a first argument. i tend to think this should be the case. the text here suggests only the parameters are provided.